public_html/wp-content/plugins/wordcamp-payments/bootstrap.php

```diff
@@ -23,6 +23,15 @@
 
 	$GLOBALS['wordcamp_budgets']    = new WordCamp_Budgets();
 	$GLOBALS['wcp_payment_request'] = new WCP_Payment_Request();
+
+	/**
+	 * Determines if the current site is a "next generation" WordCamp.
+	 *
+	 * @return bool Returns true if the current site is a "next generation" WordCamp, otherwise false.
+	 */
+	function is_next_gen_wordcamp() {
```

Comment:
As noted somewhere else in a different comment. Let's move this somewhere else. 

I think:https://github.com/WordPress/wordcamp.org/blob/production/public_html/wp-content/mu-plugins/3-helpers-misc.php is a good candidate.

Reply:
Updated in 294127b124dc48fcdf593169ca0da166922206d5